client/app/components/BottomTabs.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { withNavigation } from 'react-navigation';
+
+import connectToRedux from '../redux/lib/connectToRedux';
+import sp from '../redux/lib/stateProperties';
+
+import styles, { colors } from '../config/styles';
+import SimpleLineIcons from '@expo/vector-icons/SimpleLineIcons';
+import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+
+class BottomTabs extends Component {
+
+    constructor(props) {
+        super(props);
+        console.log(this.props.navigation)
```

Comment:
Oh totally my bad, you can delete this line `console.log(this.props.navigation)`. It's a trash print statement from before when the `navigation` prop wasn't getting passed in successfully.